installer/package_list.json

```diff
@@ -39,32 +39,30 @@
                 "photon-release",
                 "sed", "grep", "cpio", "gzip", "nano", "db", "libsolv", "libgpg-error", "hawkey", "libassuan", "gpgme", "librepo", "tdnf",
                 "libdnet", "xerces-c", "xml-security-c", "libmspack", "open-vm-tools",
-                "docker",
+                "docker", "rkt",
```

Comment:
No problem at all Touseef, I'll make those changes today.

Sent from my iPhone

> On 19 May 2015, at 4:15 am, Touseef Liaqat <notifications@github.com> wrote:
> 
> In installer/package_list.json:
> 
> > @@ -39,32 +39,30 @@
> >                  "photon-release",
> >                  "sed", "grep", "cpio", "gzip", "nano", "db", "libsolv", "libgpg-error", "hawkey", "libassuan", "gpgme", "librepo", "tdnf",
> >                  "libdnet", "xerces-c", "xml-security-c", "libmspack", "open-vm-tools",
> > -                "docker",
> > +                "docker", "rkt",
> Hi Tom,
> Thanks for updating rkt.
> Can you please revert changes in package_list.json and update the pull request so that we can merge it. We would like to add rkt in minimal at later stage after we are sure that total size of minimal install is under control.
> 
> Thanks,
> Touseef
> 
> —
> Reply to this email directly or view it on GitHub.
>